src/rendering/renderengine.cpp

```diff
@@ -919,13 +937,14 @@ void RenderEngine::renderShutdownInformation(float timer, float fullTime) {
         ghoul::fontrendering::CrDirection::Down
     );
 
+    const glm::vec2 size2 = _fontShutdown->boundingBox("Press ESC again to abort");
```

Comment:
Tiny thing, but noticed this exact sting appears twice. So maybe create a variable for it to avoid problems if someone decides to change it later?